mmdet/core/hook/memory_profiler_hook.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.runner.hooks import HOOKS, Hook
+
+try:
+    import psutil
```

Comment:
This way of writing is a bit problematic. Once the first library is not installed and the second library is installed, the user will be very confused.